Tools/px_uploader.py

```diff
@@ -336,7 +336,11 @@ def __determineInterface(self):
         self.port.flushInput()
         # Set a baudrate that can not work on a real serial port
         # in that it is 233% off.
-        self.port.baudrate = self.baudrate_bootloader * 2.33
+        try:
+            self.port.baudrate = self.baudrate_bootloader * 2.33
+        except Exception as e:
+            print(str(e) + "-> check for FTDI device failed, assuming USB connection")
```

Comment:
Is this the message you'd get as an end user on Windows if upload fails?

I would not know what to do if I got this message. 

Before I entered this domain, I did not even know what FTDI was - an end user just knows they connect a USB cable. I guess what you're saying is that this uses an FTDI driver, so we should go to the driver setup and see if there is an FTDI driver that has been marked as inactive? HOw do we fix it - change to a different USB port?

Reply:
It is not expected to fail with that message, I should have described more in detail sorry:
The standard use case is a normal user without deep knowledge connecting his pixhawk via USB and flashing it. He always get's the message "non-standard baudrates are not supported on this platform -> check for FTDI device failed, assuming USB connection" under Cygwin and it flashes successful over USB. The message is only there to inform a more experienced user who expects to have windowed mode when using his FTDI device in Cygwin about it not being detected until we have a better solution.

If the message is confusing feel free to suggest a different one. The most important thing for me was to get the upload on master woring again in Cygwin and I thought it's useful to inform the user that FTDI detection in fact failed when we run into this exception.